packages/pangea-sdk/pangea/services/authn/authn.py

```diff
@@ -602,18 +1278,40 @@ def __init__(
             ):
                 super().__init__(token, config, logger_name=logger_name)
 
-            #   - path: authn::/v1/flow/reset/password
-            # https://dev.pangea.cloud/docs/api/authn#reset-password-during-signin
+            # https://pangea.cloud/docs/api/authn#password-reset
+            # - path: authn::/v1/flow/reset/password
             def password(
                 self,
                 flow_id: str,
-                password: str,
+                password: Optional[str] = None,
                 cancel: Optional[bool] = None,
-                cb_state: Optional[str] = None,
```

Comment:
@pangea-andrest it looks like /flow/reset/password got updated and no longer has cb_state or cb_code?

![Screenshot 2023-06-30 at 10 57 13 AM](https://github.com/pangeacyber/pangea-python/assets/10459856/57f645ac-4f8b-4434-99af-c33a0b698fb4)

Let me know if this is not the case and I can undo my changes here and in `models.py`